healpix_alchemy/types.py

```diff
@@ -28,6 +28,15 @@ def process_bind_param(self, value, dialect):
             value = int(value)
         return value
 
+    def to_moc(self, rangeSet, nside, index):
+        healpixSet = np.unique(np.concatenate([np.arange(start, stop, 1) for
+                                               (start, stop) in rangeSet]))
```

Comment:
This is generating a dense array of HEALPix pixel indices. This is very memory intensive. I think that `mocpy.MOC` can be initialized directly from an interval set, can't it?

Reply:
Ah, I missed that one at first since they don't explicitly say it, but it looks like you can pass a range in to the from_string method